kubernetes/services/elastic_ip_manager.yaml

```diff
@@ -1,63 +1,65 @@
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: elasticip-configmap
-data:
-  application.properties: |
-    ignite.host=ignite-alcor-service.default.svc.cluster.local
-    ignite.port=10800
-    microservices.port.service.url=http://portmanager-service.default.svc.cluster.local:9006
----
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  labels:
-    app: elasticipmanager
-  name: elasticipmanager
-spec:
-  selector:
-    matchLabels:
-      app: elasticipmanager
-  replicas: 3
-  template:
-    metadata:
-      labels:
-        app: elasticipmanager
-    spec:
-      volumes:
-        - name: elasticip-volume
-          configMap:
-            name: elasticip-configmap
-            items:
-              - key: application.properties
-                path: application.properties
-      containers:
-        - image: zhonghaolyu/repo:ipKube1
-          #- image: fwnetworking/controller:elasticip_manager-v0.3.0
-          name: elasticipmanager-web
-          imagePullPolicy: IfNotPresent
-          command: ["java", "-jar", "/app/AlcorElasticIpManager-0.1.0.jar", "--spring.config.location=/etc/elasticip/application.properties"]
-          ports:
-            - containerPort: 8080
-          volumeMounts:
-            - name: elasticip-volume
-              mountPath: /etc/elasticip
-          envFrom:
-            - configMapRef:
-                name: elasticip-configmap
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: elasticipmanager-service
-  labels:
-    name: elasticipmanager-service
-spec:
-  type: NodePort
-  ports:
-    - port: 9011
-      targetPort: 8080
-      nodePort: 30011
-  selector:
-    app: elasticipmanager
-
+apiVersion: v1
```

Comment:
@Gzure Can you please suggest what needs to be changed to support rbac in K8s deployment? 

This is related to PR #367. Thanks.

Reply:
> @Gzure Can you please suggest what needs to be changed to support rbac in K8s deployment?
> 
> This is related to PR #367. Thanks.

We don't change k8s deploy files, because of the Rbac config file already in Jar file.

Reply:
okay thanks.